src/network.js

```diff
@@ -24,6 +24,7 @@ let current = null;
  * Creates a new `Network` object.
  * @constructor
  * @param {string} networkPassphrase Network passphrase
+ * @deprecated
```

Comment:
@Akuukis something I forgot, can you please add a warning in the `use` method in this class too? Notice how I put a link to this PR too so people can have more context (https://git.io/fj9fG)

```
  static use(network) {
    console.warn(
      'Global `Network.current()` is deprecated. Please pass explicit argument instead, e.g. `new Transaction(envelope, Networks.PUBLIC) - see https://git.io/fj9fG for more info`.'
    );

    current = network;
  }
```